lib/ABI/FunctionType.cpp

```diff
@@ -197,12 +210,40 @@ class ConversionHelper {
         revng_assert(ArgumentIterator != Function.Arguments.end());
         const model::Argument &Argument = *ArgumentIterator;
 
-        /// TODO: handle stack arguments properly.
-        /// \note: different ABIs could use different stack types.
-        /// \sa: `clrcall` ABI.
+        model::StructField Field;
+        Field.Offset = CombinedStackArgumentSize;
+        Field.CustomName = Argument.CustomName;
+        Field.OriginalName = Argument.OriginalName;
+        Field.Type = Argument.Type;
+        StackArguments.Fields.insert(std::move(Field));
+
+        auto MaybeSize = Argument.Type.size();
+        revng_assert(MaybeSize.has_value() && MaybeSize.value() != 0);
+
+        // Take stack alignment into consideration.
```

Comment:
```suggestion
        // TODO: take stack alignment into consideration.
```
?

Reply:
Nope, it's closer to "this next section is here to `take stack alignment into consideration.`".

Reply:
My bad, I was in doubt since I would have expected the word align to pop up somewhere below.